.env

```diff
@@ -99,21 +99,16 @@ SECURE_MESSAGE_HOST=secure-message
 
```

Comment:
`COLL_INST_PASSWORD=password` needs to be changed to `COLL_INST_PASSWORD=secret` on line 94. This seems to fix all the failing acceptance tests when running frontstage in docker.